ory_mocking/Handlers/handlers.go

```diff
@@ -0,0 +1,219 @@
+package Handlers
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"time"
+
+	hydra_client "github.com/ory/hydra-client-go/v2"
+	kratos_client "github.com/ory/kratos-client-go"
+)
+
+const (
+	BROWSER_LOGIN_ID        = "test_id"
+	BROWSER_LOGIN_TYPE      = "browser"
+	OAUTH2_SUBJECT          = "test1234"
+	REDIRECT                = "test.test"
```

Comment:
Correct me if I'm wrong, but this is 2 variables `AUTHORIZATION_REDIRECT_URL` and `CONSENT_REDIRECT_URL`.

Reply:
fixed